src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -425,6 +425,17 @@ public bool ShouldVersionOldDocument(DocumentsOperationContext context, Document
                     return false;
                 }
 
+                if (DatabaseRecord.GetSupportedFeaturesFor(_database.SupportedFeatures).ThrowRevisionKeyTooBigFix &&
```

Comment:
would like to have something like `_database.SupportedFeatures.ThrowRevisionKeyTooBigFix`

Reply:
done